source/renderer/app/components/widgets/splash/Splash.js

```diff
@@ -32,13 +34,14 @@ export default class SplashNetwork extends Component<Props> {
       description,
       buttonLabel,
       linkLabel,
+      currentTheme,
       backgroundImage,
     } = this.props;
 
     return (
       <div className={styles.component}>
         <div className={styles.backgroundContainer}>
-          {backgroundImage && (
+          {currentTheme === THEMES.INCENTIVIZED_TESTNET && backgroundImage && (
```

Comment:
@DeeJayElly please add this check as a variable in the Profile store: `@observable isIncentivizedTestnetTheme = currentTheme === THEMES.INCENTIVIZED_TESTNET;`